### PR TITLE
adds bottom margin to video in vf-content

### PR DESCRIPTION
**Pull request by sturobson**

### CODE REVIEW

File: components/vf-content/vf-content.scss [hunk 1]
```diff
@@ -135,6 +135,11 @@
     color: map-get($vf-colors-map, vf-color--grey);
     font-style: italic;
   }
+
+  .vf-video {
```
Comment (khawkins98) [inline components/vf-content/vf-content.scss:128]: Might be overengineering, but should we do `.vf-video + p` as this is to be used with vf-content where we wouldn't want extra spacing at the end of section ... or before and H2/H3, we wouldn't want space there, right?

Oh, and could you drop something in the changelog about this?

Comment (sturobson) [reply components/vf-content/vf-content.scss:128]: ah, you've hit on something -- I guess we should cover H2/3/4/5/6 too an lists and alter accordingly.

I'll have a play and see what's a nice solution to resolve it.

Comment (sturobson) [reply components/vf-content/vf-content.scss:128]: … because headings have top margin as well as bottom margin (if I recall correctly)

Comment (khawkins98) [reply components/vf-content/vf-content.scss:128]: 👍

**Comment by khawkins98** [comment]
Approving. I'm indifferent on if we should tackle the spacing now or as part of a more holistic thing.

**Review by khawkins98 — APPROVED** [review]

**Comment by sturobson** [comment]
I have added a little bit of 'tidying up' too.

This PR also now:

- removes `display: block` from `H1-6` and `p` elements as that is their browser default
- moved `margin-bottom` changes per element to the `@include set-type` using `$custom-margin-bottom`

**Pull request merged**